repository-data/webfiles/src/main/resources/site/freemarker/common/service/service-article.ftl

```diff
@@ -62,28 +63,27 @@
 
 
         <div class="grid-row">
-
-            <#if renderNav>
-            <div class="column column--one-third page-block page-block--sidebar article-section-nav-outer-wrapper">
-                <!-- start sticky-nav -->
-                <div id="sticky-nav">
-
-                    <#assign links = [{ "url": "#top", "title": "Top of page" }] />
-                    <#if document.latestNews?has_content >
-                          <#assign links += [{ "url": "#related-articles-latest-news-${idsuffix}", "title": 'Latest news' }] />
-                    </#if>
-                    <#assign links += getStickySectionNavLinks({ "document": document, "childPages": childPages, "includeTopLink": false }) />
-                    <#if document.relatedEvents?has_content >
-                          <#assign links += [{ "url": "#related-articles-events-${idsuffix}", "title": 'Forthcoming events' }] />
-                    </#if>
-                    <@stickyNavSections links=links></@stickyNavSections>
-                </div>
-                <!-- end sticky-nav -->
-                <#-- Restore the bundle -->
-                <@hst.setBundle basename="rb.generic.headers,publicationsystem.headers"/>
-            </div>
+            <#if navStatus == "withNav">
+                <#if renderNav>
+                    <div class="column column--one-third page-block page-block--sidebar article-section-nav-outer-wrapper">
+                        <!-- start sticky-nav -->
+                        <div id="sticky-nav">
+                            <#assign links = [{ "url": "#top", "title": "Top of page" }] />
+                            <#if document.latestNews?has_content >
+                                <#assign links += [{ "url": "#related-articles-latest-news-${idsuffix}", "title": 'Latest news' }] />
+                            </#if>
+                            <#assign links += getStickySectionNavLinks({ "document": document, "childPages": childPages, "includeTopLink": false }) />
+                            <#if document.relatedEvents?has_content >
+                                <#assign links += [{ "url": "#related-articles-events-${idsuffix}", "title": 'Forthcoming events' }] />
+                            </#if>
+                            <@stickyNavSections links=links></@stickyNavSections>
+                        </div>
+                        <!-- end sticky-nav -->
+                        <#-- Restore the bundle -->
+                        <@hst.setBundle basename="rb.generic.headers,publicationsystem.headers"/>
+                    </div>
+                </#if>
             </#if>
-
             <div class="column column--two-thirds page-block page-block--main">
```

Comment:
How can the content go wide? Wouldn't you need to change this line to something like `<div class="column ${(navStatus == "withNav" || navStatus == "withoutNav")?then("column--two-thirds", "column--wide-mode")} page-block page-block--main">` .

Reply:
Hi Geoff, I believe this is your logic, that you wrote. Because there were conflicts with widemode and navigation controller.

Reply:
Hi, I notice this is service, i believe the request was with or without nav, not going wide. So the content would still stay as is but without or with navigation.